src/hooks/useConsortiumTenants/useConsortiumTenants.js

```diff
@@ -0,0 +1,54 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+  useStripes,
+} from '@folio/stripes/core';
+
+import { OKAPI_TENANT_HEADER } from '../../constants';
+
+const DEFAULT_DATA = [];
+
+const useConsortiumTenants = () => {
+  const stripes = useStripes();
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'consortium-tenants' });
+
+  const consortium = stripes?.user?.user?.consortium;
+
+  const enabled = Boolean(consortium?.centralTenantId && consortium?.id);
+
+  const {
+    isFetching,
+    isLoading,
+    data = {},
+  } = useQuery(
+    [namespace, consortium?.id],
+    async () => {
+      const api = ky.extend({
+        hooks: {
+          beforeRequest: [
+            request => {
+              request.headers.set(OKAPI_TENANT_HEADER, consortium.centralTenantId);
+            },
+          ],
+        },
+      });
+
+      return api.get(
+        `consortia/${consortium.id}/tenants`,

```

Comment:
The default limit is 10 records, we had a problem with that already, so add a limit param, please
```suggestion
        `consortia/${consortium.id}/tenants?limit=1000`,

```